VERSION

```diff
@@ -1 +1 @@
-v0.52.0-dev
\ No newline at end of file
+v0.51.1-dev
```

Comment:
```suggestion
v0.52.0-dev
```

This is not required in the master branch. Master branch will only do minor releases. The patch release is done using the corresponding `rel` branch.